website/src/components/HomepageFeatures/index.js

```diff
@@ -1,20 +1,88 @@
 import React from 'react';
 import clsx from 'clsx';
 import styles from './styles.module.css';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Link from '@docusaurus/Link';
+import FeatureGallery from './featureGallery.js';
+import WholeRowFeature from './wholeRowFeature.js';
+import Feature from './feature.js';
 
-const FeatureList = [
+const accessibleFeatures = [
   {
-    title: 'Cloud-Native',
-    Svg: require('@site/static/img/undraw_docusaurus_mountain.svg').default,
+    title: <>Accessible</>,
     description: (
       <>
-        Built and optimized for the Google Cloud Platform
+        Leveraging the <a href="https://terra.bio" target="_blank" rel="noopener noreferrer">Terra platform </a>
+        for scalable execution, secure environment, and seamless collaboration.
       </>
     ),
+    buttons: [
+      {
+        buttonLink: 'https://app.terra.bio',
+        buttonText: 'Joint Calling Workspace'
+      },
+      {
+        buttonLink: 'https://app.terra.bio/#workspaces/help-gatk/GATK-Structural-Variants-Single-Sample',
+        buttonText: 'Single-Sample Workspace'
+      }
+    ],
   },
   {
-    title: 'Scalable',
-    Svg: require('@site/static/img/undraw_docusaurus_tree.svg').default,
+    imageUrl: 'img/logo/terra.png',
```

Comment:
I think the logo spacing looks a little off. I'd suggest either centering the entire section and having the logo be in-line with the buttons, or make the logo larger.

Reply:
This part splits the row into two equal-width sections, and then aligns the content on the left section to the right of the container, and the content on the right section to left of its container. Hence, they are aligned to the center of the screen. I changed this to aligning the content to the center of their container as you suggested, and also increased the max height of Terra logo from 100px to 150px. Please let me know if it looks better.

Reply:
The spacing looks a little off still on my screen. Let's just remove the logo and make the text centered

Reply:
We could tweak css a bit to align it better, but removing is easier, removed!